library.json

```diff
@@ -12,10 +12,10 @@
     "type": "git",
     "url": "https://github.com/devyte/ESPAsyncDNSServer.git"
   },
-  "version": "1.0.0",
+  "version": "1.0.1",
   "license": "LGPL-3.0",
   "frameworks": "arduino",
-  "platforms":"espressif",
+  "platforms": ["espressif8266", "espressif32"]
```

Comment:
I've just tried to use the branch for this PR in my project (as a quick fix until this is merged) and this line appears to be missing a `,`.

```
$ pio pkg install --library "https://github.com/maxgerhardt/ESPAsyncDNSServer#patch-1"
Resolving esp32dev dependencies...
Library Manager: Installing git+https://github.com/maxgerhardt/ESPAsyncDNSServer#patch-1
git version 2.39.0
Cloning into '/home/sebastian/.platformio/.cache/tmp/pkg-installing-mfefune7'...
remote: Enumerating objects: 13, done.
remote: Counting objects: 100% (13/13), done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 13 (delta 1), reused 4 (delta 0), pack-reused 0
Receiving objects: 100% (13/13), 4.69 KiB | 4.69 MiB/s, done.
Resolving deltas: 100% (1/1), done.
Library Manager: Could not parse manifest -> Expecting ',' delimiter: line 19 column 3 (char 424)
Library Manager: Could not parse manifest -> Expecting ',' delimiter: line 19 column 3 (char 424)
Library Manager: ESP AsyncDNSServer@1.0.1+sha.a01af6d has been installed!
Library Manager: Could not parse manifest -> Expecting ',' delimiter: line 19 column 3 (char 424)
``` 
Changing it to ` "platforms": ["espressif8266", "espressif32"],` fixes it.

Reply:
Yep my bad, fixed in https://github.com/maxgerhardt/ESPAsyncDNSServer/commit/3ccb2d39ff0d707c86431a6c0bf1c2a399e5b7b3